### ISSUE TITLE
Flex aliases no longer working with Composer

### ISSUE THREAD

**Issue by MrFjords** [symfony/flex#306]
I created a new app a couple days ago and `composer require <aliase> --dev` still works on it.

But on the project I created an hour ago the same command returns:

```sh
[InvalidArgumentException]
  Could not find package server at any version for your minimum-stability (stable). Check the package spelling or your minimum-stability
```

here is the exact sequence of commands I ran:

```sh
composer create-project symfony/skeleton my-project
cd my-project
composer require server --dev
```

**Comment by Orkin** [comment]
I have the same probleme with : 
```
composer require profiler
composer require doctrine
```

**Comment by nicolas-grekas** [comment]
Fixed in #305

**Comment by TWithers** [comment]
Any idea when the next version will be made available? Symfony/Skeleton is not working currently because of the current bug.
